pkg/network/ebpf/kprobe_types.go

```diff
@@ -47,3 +48,11 @@ const (
 )
 
 const BatchSize = C.CONN_CLOSED_BATCH_SIZE
+
+var (
+	StaticTags = map[uint64]string{
+		C.HTTP:      "HTTP",
```

Comment:
I don't think we need/want a tag just to indicate something is HTTP. If we really want this, we can infer it on the backend.

Reply:
HTTP tag would mean that we have classified at packet layer (socket filter) the flow as http protocol.
We could rename it : "packets.proto:http"

Reply:
I understand the meaning. I don't think we have a use case for that tag right now. If we did, we would likely add it on the backend when HTTP data is attached to the connection, because that saves the cost of sending the tag alongside.